concepts/index.md

```diff
@@ -3,47 +3,56 @@ layout: single
+A *Load Balancer* is associated with an ingress protocol and port range. It balances traffic among instances in its Server Group. Optionally, you can enable health checks for a load balancer, with flexiblity to define health criteria and specify the health check endpoint.
 
-This is a catch net area from which potentially lists all infrastructure components across all of Spinnaker that you have access to. You can search for Applications, Clusters, Server Groups, Instances, Load Balancers and Security Groups by wildcard. -->
+### Security Group
 
-## Load Balancers
+A *Security Group* defines network traffic access. It is effectively a set of firewall rules defined by an IP range (CIDR) along with a communication protocol (e.g., TCP) and port range.
 
-Load Balancers route incoming traffic to associated and enabled Server Groups.
+> Learn more on the [Clusters](/concepts/clusters/) page.
 
-## Security Groups
+## Deployment Management
 
-Security Groups define traffic ingress rules for Server Groups.
+You use Spinnaker's deployment management features to construct and manage continuous delivery workflows. 
 
-## Resource Mappings from Spinnaker to Provider
+### Pipeline
 
-Spinnaker | AWS | GCE | Kubernetes | Azure | OpenStack | Cloud Foundry
----|---|---|---|---|---|---
-Server Group | ASG | MIG | Replica Set
-Load Balancer | Load Balancer | L3/L4/HTTP/internal Load Balancer | Service
-Security Group | Security Group | Firewall Rule | Ingress Rule
+![](pipelines.png)
+
+*Pipelines* are the key deployment management construct in Spinnaker. They consist of a sequence of actions, known as stages. You can pass parameters from stage to stage along the pipeline. You can start a pipeline manually, or you can configure it to be started by automatic triggering events, such as a Jenkins job completing, a new Docker image appearing in your registry, a CRON schedule, or a stage in another pipeline. You can configure the pipeline to emit notifications to interested parties at various points during pipeline execution (such as on pipeline start/complete/fail), by email, SMS or HipChat.
+
+### Stage
+
+A *Stage* in Spinnaker is an action that forms an atomic building block for a pipeline. You can sequence stages in a Pipeline in any order, though some stage sequences may be more common than others. Spinnaker comes pre-packaged with a number of stages such as Deploy, Resize, Disable, Manual Judgment, and many more. You can see the full list of stages and read about implementation details for each provider in the [Reference](/reference/providers) section.
+
+### Deployment Strategies
+
+![](deployment-strategies.png)
+
+Spinnaker treats cloud-native deployment strategies as first class constructs, handling the underlying orchestration such as verifying health checks, disabling old server groups and enabling new server groups. Spinnaker supports the red/black (a.k.a. blue/green) strategy, with rolling red/black and canary strategies in active development.
```

Comment:
Not sure how I feel about "_cloud-native_ deployment strategies" - there's nothing specific to the cloud that makes these work, it just makes it easier.